tests/drivers/build_all/input/it82xx2_evb.overlay

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright (c) 2024 ITE Corporation. All Rights Reserved.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/delete-node/ &kbd;
+
+it8801_i2c: &i2c4 {
+	status = "okay";
+};
+
+#include <ite/it8801-common-cfg.dtsi>
+
+&it8801_i2c {
+	it8801_mfd: it8801@38 {
+		compatible = "ite,it8801-mfd";
+		reg = <0x38>;
+		irq-gpios = <&gpioa 1 0>;
+		#address-cells = <1>;
+		#size-cells = <1>;
+
+		ioex_it8801_kbd: it8801_kbd@40  {
+			compatible = "ite,it8801-kbd";
+			status = "okay";
+			mfdctrl = <&kso18_gp01_default &kso20_gp23_default>;
+			kso-mapping = <0 1 20 3 4 5 6
+				       17 18 16 15 11 12>;
+			row-size = <8>;
+			col-size = <13>;
+
+			kscan_input: kscan-input {
+				compatible = "zephyr,kscan-input";
+			};
+		};
+	};
+};
```

Comment:
no need to replicate the whole lot, just the overridden properties, in this case I think

```
&it8801_mfd {
        irq-gpios = <&gpioa 1 0>;

        ioex_it8801_kbd: it8801_kbd@40  {
                status = "okay";
                mfdctrl = <&kso18_gp01_default &kso20_gp23_default>;
                kso-mapping = <0 1 20 3 4 5 6 17 18 16 15 11 12>;
                row-size = <8>;
                col-size = <13>;

                kscan_input: kscan-input {
                        compatible = "zephyr,kscan-input";
                };
        };
};
```

same for the other tests

Reply:
Done.